From 924fe9aefaac006b0683c48fc5e1ca622828c53c Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Tue, 15 Dec 2015 05:44:18 -0500
Subject: [PATCH] nuplayer: Avoid crash when codec fails to load

 * Remove unnecessary check condition.

Change-Id: I3267a0c13165d74a2ea90333b42f000b51ace98b
(cherry picked from commit d10225eff7ddbc73f51c8d539c57a828b972fc42)
(cherry picked from commit d1502f80dab2793fc9977e1c55e375ee37e71722)
---
 media/libmediaplayerservice/nuplayer/NuPlayer.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/media/libmediaplayerservice/nuplayer/NuPlayer.cpp b/media/libmediaplayerservice/nuplayer/NuPlayer.cpp
index 9ae7ddbe47..fad248df28 100644
--- a/media/libmediaplayerservice/nuplayer/NuPlayer.cpp
+++ b/media/libmediaplayerservice/nuplayer/NuPlayer.cpp
@@ -2376,9 +2376,6 @@ void NuPlayer::performDecoderFlush(FlushCommand audio, FlushCommand video) {
 void NuPlayer::performReset() {
     ALOGV("performReset");
 
-    CHECK(mAudioDecoder == NULL);
-    CHECK(mVideoDecoder == NULL);
-
     updatePlaybackTimer(true /* stopping */, "performReset");
     updateRebufferingTimer(true /* stopping */, true /* exiting */);
 
